xen/xenops_interface.ml

```diff
@@ -433,6 +433,8 @@ module Vif = struct
 
 	type id = string * string
 
+	type static_ip_setting = (string * string) list
```

Comment:
Yes, it will be `Vif.Static` (because it is in the `Vif` module). Take a look at how the `locking_mode` is used:
- in xen-api: https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/xapi_xenops.ml#L534
- in xenopsd: https://github.com/xapi-project/xenopsd/blob/master/xc/xenops_server_xen.ml#L2226